README.md

```diff
@@ -1,46 +1,101 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# HTML & CSS3 & JavaScript Course
 
-# My Portfolio
+If you are not familiar with linters, read [root level README](../README.md).
 
-Hello there,
-this project has a desktop version now,
-and a responsive layout
+## Set-up GitHub Actions
 
-## Built With
+Please do the following **steps in this order**:
 
-- HTML & CSS
-- No Frameworks Used
-- No Technologies used
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
```

Comment:
- [ ] You changed your README file with this new README. It looks like it belongs to linters. Please make sure to change this README file with a professional one. I would suggest you copy the README file from your main branch and replace this file.